dashboard/test/factories/factories.rb

```diff
@@ -384,7 +384,7 @@
     association :user
     email {''}
     hashed_email {''}
-    credential_type {'email'}
+    credential_type {AuthenticationOption::EMAIL}
```

Comment:
I know this isn't from this PR, but we don't need the `{}` around `AuthenticationOption::EMAIL` since it isn't a calculated property on the factory

Reply:
I was wondering :grin: I'm going to save this for a factory cleanup PR.